eng/common/testproxy/docker-start-proxy.ps1

```diff
@@ -29,6 +35,11 @@ $SELECTED_IMAGE_TAG = "1147815"
 $CONTAINER_NAME = "ambitious_azsdk_test_proxy"
 $LINUX_IMAGE_SOURCE = "azsdkengsys.azurecr.io/engsys/testproxy-lin:${SELECTED_IMAGE_TAG}"
 $WINDOWS_IMAGE_SOURCE = "azsdkengsys.azurecr.io/engsys/testproxy-win:${SELECTED_IMAGE_TAG}"
+
```

Comment:
It looks like your changes might have had a bad merge or rebase as these shouldn't be part of your change but were recently committed to main.